playbooks/operationalizing/cloudforms_networking.adoc

```diff
@@ -0,0 +1,105 @@
+* Additional network firewall ports to open on master instances
+
+The purpose of this document is to provide an alternative solution for integrating CloudForms and OpenShift by leveraging a Local Traffic Manager (LTM), such as F5 Big-IP(™), to provide the mechanisms of routing traffic based on service type to the underlying components.
+
+== Solution Overview
+
+To overcome the requirement of deploying a router on OpenShift master instances, the provisioning and configuration of a VIP using an LTM alongside configurations in the OpenShift environment will be required. 
+
+image::images/ose-cloudforms-network.png[title=OpenShift/CloudForms Networking Overview]
+
+The following sections detail the necessary steps.
+
+
+=== Local Traffic Manager Configuration
+
+Since CloudForms requires a single point of entry for container providers, a Local Traffic Manager can be utilized to create a VIP backed by a set of virtual servers and corresponding load balancing pools representing components within OpenShift to control the flow of traffic. A VIP contains a single DNS that contains two virtual servers in the LTM. A virtual server is represented by an IP address and a service (port in this case) that will receive traffic for further processing. These include:
+
+* ose.example.com listening on port 8443 to process traffic destined for the OpenShift API
+* ose.example.com listening on port 5000 to process traffic destined for the OpenShift Metrics components (Hawkular)
+
+Also configured in the LTM are two sets of load balancing pools. Load balancing pools are logical sets of devices that are grouped together to receive and process traffic. These include:
+
+* The addresses of each OpenShift master instance to pass traffic on port 8443 for the API
+* THe addresses of each OpenShift node where each integrated HAProxy router is running on port 443. 
+
+Once these two components have been defined, the virtual server associated to port 8443 can include the load balancing pool containing the set of master instances on port 8443. The virtual server associated to port 5000 can include the load balancing pool containing the set of node instance that contain running instances of the integrated HAProxy router to pass traffic to the Hawkular metrics services within the OpenShift cluster.
+
+
+==== SSL Considerations
+
+To provide the greatest level of flexibility, SSL can be configured in several places depending on the architecture of the environment. SSL termination for the API can occur at eiter the LTM or OpenShift level. For the metrics component, SSL termination can either occur at the LTM layer, Hawkular layer or at the HAProxy layer with the help of SSL reencryption support. The following links provide examples of how each type are configured:
```

Comment:
It seems like somewhere in here (or perhaps elswhere in this doc) there is an opportunity to reference theplaybooks Load Balancing Guide.